Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -178,9 +177,7 @@ bool Connection::processMessage()
     Vector<AttachmentInfo> attachmentInfo(attachmentCount);
 
     if (attachmentCount) {
-        memcpy(static_cast<void*>(attachmentInfo.data()), messageData, sizeof(AttachmentInfo) * attachmentCount);
-        messageData += sizeof(AttachmentInfo) * attachmentCount;
-
+        memcpySpan(asMutableByteSpan(attachmentInfo), consumeSpan(messageData, sizeof(AttachmentInfo) * attachmentCount));
```

Comment:
Oh, calling `asMutableByteSpan()` is the right thing to do here. I've just noticed that `attachmentInfo` is a Vector.

Reply:
You likely want to do `asMutableByteSpan(attachmentInfo.mutableSpan())`